DMTR-91.tex

```diff
@@ -1,564 +1,1334 @@
-LVV-T216 & Conditional Pass & {\textit Missing git-lfs in step 2 and 3. Used docker image instead. Remaining steps pass.} \\\hline
-LVV-T217 & Conditional Pass & {\textit Missing git-lfs in step 2 and 3. Used docker image instead. Remaining steps pass.} \\\hline
-LVV-T218 & Conditional Pass & {\textit Missing git-lfs in step 2 and 3. Used docker image instead. Last step result deviated slightly from the expected result for one consumer.} \\\hline
+    \\\hline
+
+\caption{Test Results Summary}
+\label{table:summary}
 \end{longtable}
 
 \subsection{Overall Assessment}
 \label{sect:overallassessment}
 
-The test campaign is considered in overall: PASS.
-
-Missing elements on the Kubernetes Commons prevented us from building the Docker images in place and using a complete night of unique alerts.
-However, using pre-built Docker images with a smaller, repeated sample of alerts enabled the test to proceed as expected.
+The test campaign is considered in overall:
+PASS.\\[2\baselineskip]Missing elements on the Kubernetes Commons
+prevented us from building the Docker images in place and using a
+complete night of unique alerts. However, using pre-built Docker images
+with a smaller, repeated sample of alerts enabled the test to proceed as
+expected.
 
 
 \subsection{Recommended Improvements}
 \label{sect:recommendations}
 
-Re-execution with the missing components on the Kubernetes Commons.
+Re-execution with the missing components on the Kubernetes
+Commons.\\[2\baselineskip]Examination of the deserialized alerts in the
+logs is cumbersome and does not allow real verification of the alert
+content. Improved tools for counting alerts, comparing their contents to
+the sent alerts, and timing throughput would all improve the utility of
+this test.
 
-Examination of the deserialized alerts in the logs is cumbersome and does not allow real verification of the alert content. 
-Improved tools for counting alerts, comparing their contents to the sent alerts, and timing throughput would all improve the utility of this test.
 
+\newpage
+\section{Detailed Test Results}
+\label{sect:detailedtestresults}
 
 
+  \subsection{Test Cycle LVV-C3 }
 
-\newpage
+Open test cycle {\it \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testrun/LVV-C3}{LDM-503-5: Alert Distribution Validation}} in Jira.
 
-\section{Detailed Test Results}
-\label{sect:detailed}
+  LDM-503-5: Alert Distribution Validation\\
+  Status: Done
+
+  Test run with test cases in draft.
+
+
+  \subsubsection{Software Version/Baseline}
+    This test uses only the code in\\
+https://github.com/lsst-dm/alert\_stream/ , commit
+ef3c9e16c42a3265d4f6258ba6983be8b009e7c0
+
+
+  \subsubsection{Configuration}
+    \textbf{Documents}\\[2\baselineskip]This test report refers to the
+execution of tests described in the section 1.2 and documented in the
+\citeds{LDM-533} test specification issue
+2.0.\\[2\baselineskip]\textbf{Hardware}\\[2\baselineskip]This test
```

Comment:
Where is LDM-533 v2? Even the latest version I can find (https://ldm-533.lsst.io/v/DM-16743/index.html, from today) only claims to be version 1 + some extra commits.

Reply:
The Test Specification LDM-533 should have been approved before the test ware executed.